backend/src/shared/interfaces/SlideObjectFilter.ts

```diff
@@ -0,0 +1,4 @@
+export class SlideObjectFilter {
```

Comment:
Shouldnt be there an interface? There is possibility to use class but for rather for dto with using decorators from class-validator: 
```
export class CreateUserDto {
  @IsEmail()
  @IsNotEmpty()
  @Length(6, 30)
  email: string;
```

Reply:
I get this error when I use an interface:

![image](https://github.com/user-attachments/assets/69fbe36b-da37-4387-921f-b83983c1733a)

Reply:
I get this error when I use an interface:

![image](https://github.com/user-attachments/assets/20893e5b-a826-4aa9-9958-f42edba624e4)